package.json

```diff
@@ -121,15 +121,15 @@
     "test-docker-integration": "env-cmd $npm_package_options_env_cmd npm run test-integration",
     "docs": "esdoc && cp docs/ROUTER esdoc/ROUTER && cp docs/favicon.ico esdoc/favicon.ico",
     "teaser": "node scripts/teaser",
-    "test-unit": "mocha --require scripts/mocha-bootload --globals setImmediate,clearImmediate --exit --check-leaks --colors -t 30000 --reporter spec \"test/unit/**/*.js\"",
```

Comment:
is this related to the issue? or is it just not needed anymore?

Reply:
Oh, I didn't really mean to commit this but no, this isn't needed anymore.

Reply:
maybe we can put this in a separate pr then so if it would cause trouble at some end it's easier to pin it down using bisect